README.md

```diff
@@ -307,6 +307,17 @@ all four button states ("default", "default + hover", "casting", "casting + hove
 folder contains grey, black, and blue versions of the icons.
 
 
+### Events
+
+#### `chromecastConnected`: Triggers when Chromecast connected
```

Comment:
Could we make each of these bullet points rather than headings? That will give us more space for the description if necessary.

Reply:
Fixed using bullet points, looks much cleaner!